packages/storefront-lwc/package.json

```diff
@@ -66,26 +70,39 @@
     "concurrently": "^5.0.0",
     "cpx": "^1.5.0",
     "cross-env": "^7.0.2",
+    "esm": "^3.2.25",
```

Comment:
We might need to check `esm` for security

Reply:
@echessman